bin/process-csv.js

```diff
@@ -31,143 +31,163 @@
 
     // Generate "products.csv" based on the source data.
     processProducts = function (workPath, targetPath) {
-        var skippedRecords = [];
+        var skippedRecords = [],
+            authors = {};
         console.log('Processing CSV data with workPath="%s", targetPath="%s"', workPath, targetPath);
         fs.readdir(targetPath + '/images', function (err, imageFilenames) {
             if (err) {
                 console.error('Could not read images path "%s", please check the path exists and try again', targetPath + '/images');
                 return;
             }
-            fs.createReadStream(workPath + '/Books.csv')
+            fs.createReadStream(workPath + '/Authors.csv')
                 .pipe(csv.parse({ columns: true }))
-                .pipe(csv.transform(function (record) {
-                    var title, imageFilename;
-
-                    if (!record.RRP) {
-                        skippedRecords.push({ record: record, reason: 'RRP is zero or missing' });
-                        return undefined;
+                .pipe(csv.transform(function (author) {
+                    var authorName = (author.AuthFirstName + ' ' + author.AuthLastName).trim();
```

Comment:
There is a middle name field, but it isn't used except for one value, which is "Archdiocese of Perth".